js/packages/core/src/ufg/types.ts

```diff
@@ -11,13 +11,22 @@ import {
   type DomSnapshot,
   type AndroidSnapshot,
   type IOSSnapshot,
+  RendererSettings,
 } from '@applitools/ufg-client'
 
 export * from '../automation/types'
 
 export type SnapshotTarget = MaybeArray<DomSnapshot> | MaybeArray<AndroidSnapshot> | MaybeArray<IOSSnapshot>
 export type Target<TSpec extends SpecType> = SnapshotTarget | AutomationCore.Target<TSpec>
 
+type RendererWithProps = Renderer & {
+  properties?: BaseCore.CustomProperty[]
+}
+
+type RendererSettingsWithProps = RendererSettings & {
+  renderers: RendererWithProps[]
+}
```

Comment:
I think this is fine, but patching one type and then patching another with it, seems unnecessary. Also the name `RendererSettingsWithProps` is confusing, because we don't really need both, we will need them separately. This is why I suggest doing the `GetBaseEyesSettings` with properties, without patching the `renderers` type.

```suggestion
type GetBaseEyesSettings = RendererSettings & {
  properties?: BaseCore.CustomProperty[]
}
```